From b2ea44969c5e51a5809622384728859d7f3a2b8a Mon Sep 17 00:00:00 2001
From: Sandor Yu <Sandor.yu@nxp.com>
Date: Fri, 19 Jun 2020 16:25:51 +0800
Subject: [PATCH 32/49] LF-1516: drm: cdns-mhdp: fix error check variable name
 for clk_pxl_link

fix error check variable name for clk_pxl_link.
Report by Coverity ID:6652947

Signed-off-by: Sandor Yu <Sandor.yu@nxp.com>
Reviewed-by: Fancy Fang <chen.fang@nxp.com>
---
 drivers/gpu/drm/imx/mhdp/cdns-mhdp-imx8qm.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/drivers/gpu/drm/imx/mhdp/cdns-mhdp-imx8qm.c b/drivers/gpu/drm/imx/mhdp/cdns-mhdp-imx8qm.c
index a3ba3da4b05d..2ee4e8748b77 100644
--- a/drivers/gpu/drm/imx/mhdp/cdns-mhdp-imx8qm.c
+++ b/drivers/gpu/drm/imx/mhdp/cdns-mhdp-imx8qm.c
@@ -167,7 +167,7 @@ int imx8qm_clocks_init(struct imx_mhdp_device *imx_mhdp)
 	}
 
 	clks->clk_pxl_link = devm_clk_get(dev, "clk_pxl_link");
-	if (IS_ERR(clks->clk_pxl_mux)) {
+	if (IS_ERR(clks->clk_pxl_link)) {
 		dev_warn(dev, "failed to get pxl link clk\n");
 		return PTR_ERR(clks->clk_pxl_link);
 	}
-- 
2.29.2

